lib/fabrik/DirectorInstance.js

```diff
@@ -318,6 +320,32 @@ class DirectorInstance extends BaseInstance {
         }
       });
   }
+
+  scheduleAutoUpdate() {
+    const options = {
+      instance_id: this.guid,
+      repeatInterval: CONST.SCHEDULE.RANDOM,
+      timeZone: _.get(config, 'scheduler.jobs.service_instance_update.time_zone', 'UTC')
+    };
+    return utils
+      .retry(tries => {
+        logger.info(`+-> ${CONST.ORDINALS[tries]} attempt to schedule auto update for : ${this.guid}`);
+        if (utils.isFeatureEnabled(CONST.FEATURE.SCHEDULED_UPDATE)) {
+          return this
+            .serviceFabrikClient
+            .scheduleUpdate(options)
+            .catch(err => {
+              logger.error(`Error occurred while setting up auto update for : ${this.guid}`, err);
+              throw err;
```

Comment:
Will this declare service instance creation as failed? By this time all operations related to instance creation is done e.g. entry in bosh, security group in cf etc.

Reply:
Good catch. Fixing this by logging the error after all retries fail. Also this is done in tap so it will not change the outcome of create operation.